includes/wc-internal/orders.php

```diff
@@ -31,7 +31,8 @@ public function createOrder(string $orderId)
         return;
       $trunkrsOrder = new TRUNKRS_WC_Order($orderId);
 
-      if (!$trunkrsOrder->isTrunkrsOrder)
+      if (!$trunkrsOrder->isTrunkrsOrder
```

Comment:
Maybe you could change this to an or instead:

So when you either have enabled the all orders for trunkrs or it is a Trunkrs order we will announce it. 🙂